resources/startstopserver.ts

```diff
@@ -20,63 +20,64 @@ import { ECSClient, UpdateServiceCommand } from '@aws-sdk/client-ecs';
 import { APIGatewayEvent, APIGatewayProxyHandler, Context } from 'aws-lambda';
 
 //Set the AWS Region
-const REGION = process.env.REGION; 
+const REGION = process.env.REGION;
 
-const SERVICE_NAME = process.env.SERVICE_NAME;  
-const CLUSTER_ARN = process.env.CLUSTER_ARN; 
+const SERVICE_NAME = process.env.SERVICE_NAME;
+const CLUSTER_ARN = process.env.CLUSTER_ARN;
 const PASSWORD = process.env.PASSWORD;
 
-const handler: APIGatewayProxyHandler = async (event: APIGatewayEvent, context: Context) => {
-    console.log("request: " + JSON.stringify(event));
-    let responseCode = 400;
-    let message = "authentication failed";
+export const handler: APIGatewayProxyHandler = async (event: APIGatewayEvent, context: Context) => {
```

Comment:
Same as before, wrt specifying the return type: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L140-L154